src/coverlet.collector/version.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+    "version": "1.0.0",
```

Comment:
This is using a fixed version of 1.0.0. If you want NB.GV to work, you should specify either `1.0` or `1.0.0-build.{height}`

Reply:
Did it, but no luck 
![image](https://user-images.githubusercontent.com/7894084/58367326-3b0c1d80-7ede-11e9-89bf-9820f2832c76.png)